src/NuGet.Clients/VsExtension/NuGet.Tools.csproj

```diff
@@ -36,8 +37,10 @@
     <StartArguments>/rootsuffix Exp /log</StartArguments>
     <CreateVsixContainer>True</CreateVsixContainer>
     <DeployExtension>True</DeployExtension>
+    <NoWarn>1762;649</NoWarn>
```

Comment:
>NoWarn [](start = 5, length = 6)

Is 649 this one?
[https://msdn.microsoft.com/en-us/library/03b5270t.aspx](https://msdn.microsoft.com/en-us/library/03b5270t.aspx)

I don't think we should ignore it then.